src/dscim/menu/main_recipe.py

```diff
@@ -318,8 +323,9 @@ def scc():
             self.logger.info("Processing SCC calculation ...")
             if self.fit_type == "quantreg":
                 self.full_uncertainty_iqr
-                self.calculate_scc
-                self.stat_uncertainty_iqr
+                if len(self.fair_aggregation) > 0:
+                    self.calculate_scc
+                    self.stat_uncertainty_iqr
```

Comment:
Ideally, we add coverage for the new lines in this PR. Can you do that if you have time, @JMGilbert ? It can be a new PR